renovate.json

```diff
@@ -1,6 +1,5 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": [
-    "local>freckle/renovate-config"
-  ]
+  "extends": ["local>freckle/renovate-config"],
+  "minimumReleaseAge": "5 days"
```

Comment:
@StevenXL we don't need this as this is a library, can you revert back to the original Renovate commit?

For context on why, consider this example:

1. Renovate updates a `lib-a` dep from v1 to v2 and auto-merges
2. `lib-a` auto-releases v1.1
4. Renovate updates the `app-b` dep `lib-a` from v1.0 to v1.1 and auto-merges
5. `app-b` is auto-released

Since we will wait 5 days at (3), we do not need to also wait 5 days at (1).